pkg/media/samplebuilder/sampleSequenceLocation.go

```diff
@@ -0,0 +1,148 @@
+// Package samplebuilder provides functionality to reconstruct media frames from RTP packets.
+package samplebuilder
+
+import "math"
+
+type sampleSequenceLocation struct {
+	// head is the first packet in a sequence
+	head uint16
+	// tail is always set to one after the final sequence number,
+	// so if head == tail then the sequence is empty
+	tail uint16
+}
+
+func (l sampleSequenceLocation) empty() bool {
+	return l.head == l.tail
+}
+
+func (l sampleSequenceLocation) hasData() bool {
+	return l.head != l.tail
+}
+
+func (l sampleSequenceLocation) count() uint16 {
+	return seqnumDistance(l.head, l.tail)
+}
+
+/*
+func (l sampleSequenceLocation) adjacent(after sampleSequenceLocation) bool {
+	if !l.hasData() {
+		return false
+	}
+	if !after.hasData() {
+		return false
+	}
+	return l.tail == after.head
+}
+
+func (l sampleSequenceLocation) reset() {
+	l.head, l.tail = 0, 0
+}
+
+func (l sampleSequenceLocation) flush() {
+	l.tail = l.head
+}
+*/
```

Comment:
cleanup?

Reply:
I implemented them while it was fresh in my brain thinking I might need them but lint forces them to be used and they aren't needed, so we can delete them I suppose.

Reply:
Done